addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -259,54 +259,6 @@
             </field>
         </record>
 
-        <record id="product_product_expense_form_view" model="ir.ui.view">
```

Comment:
i think we have to keep this view. because it's not inherited .. if you remove it then default variant view display which contain some other ununsed field which is not usefule for expense may be..